benchmark/benchmark.js

```diff
@@ -77,7 +103,62 @@ suite
       xs: ['foo', 'bar']
     })
   })
-  .on('cycle', function (event) {
-    console.log(String(event.target))
+  .add('read request body JSON', async function () {
```

Comment:
remove async

Reply:
I think we can't remove async

https://github.com/tinylibs/tinybench/blob/v2.5.0/src/utils.ts#L31